test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -529,7 +532,8 @@
       (tk-app/stop app))))
 
 (deftest large-request-test
-  (testing (str "request to Jetty fails with a 413 error if the request header "
+  ;; This changed from 413 to 431 in https://github.com/eclipse/jetty.project/commit/e53ea55f480a959a2f1f5e2dbdbfc689d61c94a6
+  (testing (str "request to Jetty fails with a 431 error if the request header "
```

Comment:
Would be good to update the expected HTTP status code change for this in the [request-header-max-size](https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/blob/1.7.0/doc/jetty-config.md#request-header-max-size) docs.